maven-release-manager/src/main/java/org/apache/maven/shared/release/phase/AbstractMapVersionsPhase.java

```diff
@@ -308,9 +309,8 @@ else if ( releaseDescriptor.isBranchCreation() )
                     {
                         messageKey = getMapversionPromptKey( releaseDescriptor );
                     }
-                    String message =
-                            MessageFormat.format( resourceBundle.getString( messageKey ), project.getName(),
-                                    projectId );
+                    String message = MessageFormat.format( resourceBundle.getString( messageKey ), project.getName(),
```

Comment:
I wonder whether those bundles were of use at all during the last couple of years. I consider then pointless because other messages are hardcoded as well.

Reply:
this can be a simplification for later, why not

Reply:
I prefer it. It never has got traction.

Reply:
https://issues.apache.org/jira/browse/MRELEASE-1098

Reply:
done